src/actions/types.ts

```diff
@@ -70,6 +70,13 @@ export type DelegateActionParams = {
   delegator?: Address
 }
 
+export type VoteActionParams = {
+  chainId: number
+  governorAddress?: Address
+  proposalId?: bigint | FilterOperator
+  support?: bigint | FilterOperator
```

Comment:
It feels undemocratic to let people reward others who vote a specific way... should we support this?